en/django_views/README.md

```diff
@@ -33,6 +33,6 @@ Another error! Read what's going on now:
 
 ![Error](images/error.png)
 
-This one is easy: *TemplateDoesNotExist*. Let's fix this bug and create a template in the next chapter!
+There is no "It works" anymore, huh? Don't worry, it's just an error page, nothing to be scared of! They're actually pretty useful.  You can read that *TemplateDoesNotExist*. This one is easy: Let's fix this bug and create a template in the next chapter!
```

Comment:
Rather than referring back to "It works" (as other stuff happened since that), we could mention that this means the server is running again now.